app/test/unit/infer-comparison-branch-test.ts

```diff
@@ -122,41 +112,14 @@ describe('inferComparisonBranch', () => {
       repo,
       branches,
       pr,
-      null,
-      mockGetRemotes,
-      comparisonCache
+      mockGetRemotes
     )
 
     expect(branch).not.toBeNull()
     expect(branch!.upstream).toBe(branches[5].upstream)
   })
 
-  it('Returns the default branch of the fork if it is ahead of the current branch', async () => {
```

Comment:
This test is not relevant anymore, since now the `origin` branch is never returned for a fork